haystack/components/evaluators/context_relevance.py

```diff
@@ -160,24 +147,23 @@ def run(self, questions: List[str], contexts: List[List[str]]) -> Dict[str, Any]
         :returns:
             A dictionary with the following outputs:
                 - `score`: Mean context relevance score over all the provided input questions.
-                - `individual_scores`: A list of context relevance scores for each input question.
-                - `results`: A list of dictionaries with `statements` and `statement_scores` for each input context.
+                - `results`: A list of dictionaries with `relevant_statements` and `score` for each input context.
         """
         result = super(ContextRelevanceEvaluator, self).run(questions=questions, contexts=contexts)
 
-        # calculate average statement relevance score per query
         for idx, res in enumerate(result["results"]):
             if res is None:
-                result["results"][idx] = {"statements": [], "statement_scores": [], "score": float("nan")}
+                result["results"][idx] = {"relevant_statements": [], "score": float("nan")}
                 continue
-            if not res["statements"]:
-                res["score"] = 0
-            else:
-                res["score"] = np_mean(res["statement_scores"])
+            if "relevant_statements" in res:
+                if res["relevant_statements"] == ["Insufficient Information"]:
```

Comment:
Could you make this more robust, just in case the LLM does not give this exact same output. When I was trying ragas, I ran into this issue (see issue description). We could make it insensitive to capitalisation and maybe strip dots.

Reply:
done 👍🏽